Src/BootCamp.Chapter/Player.cs

```diff
@@ -73,34 +58,67 @@ public Item[] GetItems(string name)
         // When a slot is equiped, it contributes to total defense
         // and total attack.
         // Implement equiping logic and total defense/attack calculation.
+        /// <summary>
+        /// Everyone can carry this much weight at least.
+        /// </summary>
+        private const int baseCarryWeight = 30;
+
+        private int Name { get; set; }
+        private int Hp { get; set; }
+
+        /// <summary>
+        /// Each point of strength allows extra 10 kg to carry.
+        /// </summary>
+        private int Strenght { get; set; }
+
+
+        /// <summary>
+        /// Needed only for the extra task.
+        /// </summary>
+        private Equipment Equipment { get; set; }
```

Comment:
I'm not sure if you're 100% trying to complete the challenge or not since you've not implemented the attack/weight checks however Equipment is never initialised so trying to call .Equip will raise a `NullReferenceException`

Reply:
No not going for the challenge just yet. i just made a start. To much work to do.